src/sentry/testutils/factories.py

```diff
@@ -321,6 +322,10 @@ def make_word(words=None):
     "platform": "python",
 }
 
+default_detector_config_data = {
```

Comment:
this stuff just showing up from the other pr cause of base branch?

Reply:
Yeah, I hadn't rebased yet, so github was probably confused